bin/phpcs-whitelist.js

```diff
@@ -0,0 +1,12 @@
+// If the file path starts with anything like in the array below, it should be linted.
+module.exports = [
+	'_inc/lib/debugger/',
+	'_inc/lib/class.jetpack-password-checker.php',
+	'extensions/',
+	'sync/class.jetpack-sync-module-auth.php',
+	'class.jetpack-gutenberg.php',
+	'class.jetpack-plan.php',
+	'modules/module-extras.php',
+	'modules/module-info.php',
+	'functions.opengraph.php',
+];
```

Comment:
I think having the whitelist in a separate file from the precommit hook script is a great idea regardless. :-)